### DISCUSSION TITLE
Is there any way to hide navigation painting when testing my game in editor?

### DISCUSSION THREAD

**Discussion by xand00** [TokisanGames/Terrain3D#378] [Questions] [answered]
When some area is navigation painted, it turns purple color, can i somehow hide this color and enable it later on when i want to paint navigation again?

**Comment by TokisanGames** [comment, marked as answer]
Click any other tool other than navigation.

**Reply by xand00** [reply]
![image](https://github.com/TokisanGames/Terrain3D/assets/32330527/916e639d-61fd-431d-8a91-694415b5e73c)
unfortunately that does not work

**Reply by TokisanGames** [reply]
Click Terrain3D. Click any other tool on the sidebar. 
The purple is only enabled if you've clicked the navigation tool, or turned on the navigation debug view in the material, which the navigation tool does.

**Reply by xand00** [reply]
Oh, my bad, i didn't understand first answer, i thought you were saying about clicking other node, thank you! :)

**Reply by TokisanGames** [reply]
You're welcome
